update readme (#366)
rust-fuzz/afl.rs: README.md

Documentation can be found in the [Rust Fuzz Book](https://rust-fuzz.github.io/book/afl.html).

## Hints

> By default the AFL++ CMPLOG feature is activated, which is an amazing feature to achieve good code coverage.

Reviewer: ```suggestion
By default, the AFL++ [CMPLOG](https://github.com/AFLplusplus/AFLplusplus/blob/stable/instrumentation/README.cmplog.md) feature is activated, which helps to achieve good code coverage.
```

> However it is not beneficial to activate CMPLOG on more than 2 instances.

Reviewer: ```suggestion
However, it is not beneficial to activate CMPLOG on more than two instances.
```

So if you run multiple AFL++ instances on your fuzzing target, you can disable CMPLOG by specifying the command line parameter '-c -'.

> It is highly recommended to familiarize yourself with the AFL++ features and how to successfully run a fuzzing campaign. This [document](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/fuzzing_in_depth.md) will help you getting started.

Reviewer: ```suggestion
This [document](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/fuzzing_in_depth.md) will familiarize you with AFL++ features to help in running a successful fuzzing campaign.
```
